### PR TITLE
train acc loss, validation acc tensorboard graphs are added 

### PR DESCRIPTION
**Pull request by YCAyca**
train loss comes from train dataset, train acc comes from train dataset but in test mode so we dont have data augmentation for this dataset and validation accuracy comes from validation dataset are calculated and logged in tensorboard during train time

### CODE REVIEW

**Comment by OpenMMLab-Assistant-007** [comment]
Hi
@YCAyca 
We are grateful for your efforts in helping improve this open-source project during your personal time. 
Welcome to join OpenMMLab Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Comment by jyothsna-phd22** [comment]
Hi, how to compute validation loss each epoch along with training loss? Pl help me if any code base... I have ur codes but val_loss computation is not there in eval_one_epoch(....). Kindly help me in this regard
